src/renate/benchmark/scenarios.py

```diff
@@ -56,7 +59,10 @@ def prepare_data(self) -> None:
     @abc.abstractmethod
```

Comment:
What does an `abc` with code do?

Reply:
Forces the implementation of the function:

```
import abc

class A(abc.ABC):
    @abc.abstractmethod
    def foo(self):
        print("parent")
        
        
class B(A):
    def foo(self):
        super().foo()
        
class C(A):
    pass

A().foo()  # Fails
B().foo()  # Works
C().foo()  # Fails
```